diff -ur ConcurrentLogHandler-0.8.4/src/cloghandler.py ConcurrentLogHandler-0.8.4.patched/src/cloghandler.py
--- ConcurrentLogHandler-0.8.4/src/cloghandler.py	2009-11-03 04:40:37.000000000 +1000
+++ ConcurrentLogHandler-0.8.4.patched/src/cloghandler.py	2012-09-20 12:34:31.259438206 +1000
@@ -191,9 +191,10 @@
         """ Release file and thread locks. Flush stream and take care of closing
         stream in 'degraded' mode. """
         try:
-            self.stream.flush()
-            if self._rotateFailed:
-                self.stream.close()
+            if not self.stream.closed:
+                self.stream.flush()
+                if self._rotateFailed:
+                    self.stream.close()
         finally:
             try:
                 unlock(self.stream_lock)
